invenio_cache/decorators.py

```diff
@@ -31,3 +32,70 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return caching
+
+
+def cached_with_expiration(f):
```

Comment:
Do you prefer to have a name that states that this is a `in-process` cache and not distributed? e.g.`in_process_cache(f)`

Reply:
I am working on adding tests.